Xamarin.Forms.GoogleMaps/Xamarin.Forms.GoogleMaps.Android/Logics/PinLogic.cs

```diff
@@ -101,7 +102,7 @@ void OnInfoWindowClick(object sender, GoogleMap.InfoWindowClickEventArgs e)
             // lookup pin
             var targetPin = LookupPin(e.Marker);
 
-            // only consider event handled if a handler is present. 
+            // only consider event handled if a handler is present.
```

Comment:
Please don't contain "not the purpose" changes.
Code cleaning is "not purpose" for this PR.